### PR TITLE
8311492: FontSmoothingType LCD produces wrong color when transparency is used

### PR DESCRIPTION
**Pull request by kevinrushforth**
Clean backport to jfx22u.

---------
### Progress
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue
- [x] [JDK-8311492](https://bugs.openjdk.org/browse/JDK-8311492) needs maintainer approval

### Issue
 * [JDK-8311492](https://bugs.openjdk.org/browse/JDK-8311492): FontSmoothingType LCD produces wrong color when transparency is used (**Bug** - P3 - Approved)

### Reviewers
 * [Phil Race](https://openjdk.org/census#prr) (@prrace - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jfx22u.git pull/10/head:pull/10` \
`$ git checkout pull/10`

Update a local copy of the PR: \
`$ git checkout pull/10` \
`$ git pull https://git.openjdk.org/jfx22u.git pull/10/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 10`

View PR using the GUI difftool: \
`$ git pr show -t 10`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jfx22u/pull/10.diff">https://git.openjdk.org/jfx22u/pull/10.diff</a>

</details>

### Webrev
[Link to Webrev Comment](https://git.openjdk.org/jfx22u/pull/10#issuecomment-1936710577)

### CODE REVIEW

**Comment by kevinrushforth** [comment]
/approval request I request to backport this to jfx22u for 22.0.1. The backport is clean, and this is a low-risk fix.

**Review by prrace — APPROVED** [review]

**Comment by kevinrushforth** [comment]
@johanvos Presuming you have no concerns, can you do the maintainer approval (`/approve yes`) when you get a chance?

**Comment by kevinrushforth** [comment]
Thanks.

/integrate

**Pull request closed without merging**
